src/pages/student/Bookmarks.js

```diff
@@ -419,11 +431,13 @@ function Bookmarks({
             )}
           </div>
         </div>
-        <div className="results-content">
-          <GridComponent
-            displayBanner={true}
-            bookmarks={student.bookmarked_clubs}
-          />
+        <div style={student.bookmarked_clubs.length > 9 ? {width: '1160px', height:'980px', overflowY: 'scroll', overflowX: 'hidden'} : null}>
```

Comment:
hello thank you satiya for working on this!! this looks awesome on desktop zoomed out but I have a couple of concerns about responsiveness. the GridComponent already has responsive styling but when we switch to mobile view/zoom in on desktop, those styles are seemingly overridden by something in this PR (see below)

current mobile:

<img width="241" alt="Screen Shot 2021-11-05 at 10 14 14 AM" src="https://user-images.githubusercontent.com/46834662/140551717-1702e4c2-914d-4981-a699-a6c6646fd4a3.png">

current desktop:

<img width="1167" alt="Screen Shot 2021-11-05 at 10 13 10 AM" src="https://user-images.githubusercontent.com/46834662/140551750-48a7478e-6b93-4c72-a6f5-bd9cac8b0f2e.png">

versus desired desktop:

<img width="994" alt="Screen Shot 2021-11-05 at 10 13 21 AM" src="https://user-images.githubusercontent.com/46834662/140551799-cbd2dac8-62aa-48be-ae9f-451d03c6f369.png">

current hypothesis is that nesting the GridComponent inside another div is the cause of this, but what do you think? @satiyak

Reply:
could we please also add in a little bit of padding to the right of the third column so the scrollbar for the y-axis is not overlapping the club card components?

<img width="884" alt="Screen Shot 2021-11-05 at 10 20 29 AM" src="https://user-images.githubusercontent.com/46834662/140552598-47fb72fd-7b12-4321-ac0a-9754499ccc09.png">